### ISSUE TITLE
Provide a CSS for the generated HTML document

### ISSUE THREAD

**Issue by xcoulon** [bytesparadise/libasciidoc#63]

**Comment by gdamore** [comment]
Should we just copy the asciidoctor CSS?  (I don't particularly love that particular CSS, but it might be what people expect.)

In order to get the output I want, I found it necessary to write my own.

In an ideal world, the default UA style sheet would be adequate to get reasonable rendering.  I think a few things -- tables and admonition block come to mind -- might make that harder.

**Comment by xcoulon** [comment]
yes, reusing the Asciidoctor CSS would be an option, but I thought we could have something a little more modern, and to be honest I also opened this issue to give me a change to dive a bit into CSS again ;)

**Comment by bwklein** [comment]
For some additional feedback on this. I use the HTML5s backend and use my own CSS for all projects.  I don't have anything with the default HTML and CSS provided by Asciidoctor.
